doc/classes/EditorPaths.xml

```diff
@@ -51,7 +51,7 @@
 		<method name="get_project_settings_dir" qualifiers="const">
 			<return type="String" />
 			<description>
-				Returns the project-specific editor settings path. Projects all have a unique subdirectory inside the settings path where project-specific editor settings are saved.
+				Returns the project-specific editor settings path, usually [code]"res://.godot/editor"[/code]. Projects all have a unique subdirectory inside the settings path where project-specific editor settings are saved.
```

Comment:
Just noticed this addition. It's nice, because it's one-of-a-kind among these methods. `res://` isn't mentioned anywhere else in this class, for obvious reasons, so maybe it's worth specifying more explicitly while we're at it?

This could be more consistent to the surrounding descriptions:
```suggestion
				Returns the relative path to the editor settings for this project. This is usually [code]"res://.godot/editor"[/code]. Projects all have a unique subdirectory inside the settings path where project-specific editor settings are saved.
```